python/CMakeLists.txt

```diff
@@ -1,32 +1,21 @@
 cmake_minimum_required(VERSION 3.17.0)
 
 project(pygbs LANGUAGES CXX)
-#
+
 # gslib
-#
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../)
-# site-package dir
-if(EXISTS $ENV{SP_DIR})
-    file(TO_CMAKE_PATH $ENV{SP_DIR} SP_DIR)
-    set(${PROJECT_NAME}_INSTALL_DIRECTORY ${SP_DIR}/${PROJECT_NAME})
-else()
-    if(UNIX)
-        set(${PROJECT_NAME}_INSTALL_DIRECTORY ${CONDA_PREFIX}/lib/python${python_version_major}.${python_version_minor}/site-packages/${PROJECT_NAME})
-    else()
-        set(${PROJECT_NAME}_INSTALL_DIRECTORY ${CONDA_PREFIX}/Lib/site-packages/${PROJECT_NAME})
-    endif()
-endif()
+include_directories(${CMAKE_SOURCE_DIR})
+
+# Set installation directories
+find_package(PythonInterp)
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "from distutils.sysconfig import get_python_lib; import os;print(get_python_lib())"
+                OUTPUT_VARIABLE PY_SP_DIR OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+set(SP_DIR "${PY_SP_DIR}" CACHE PATH "Site-package directory")
+message("  -> site-packages: " ${SP_DIR})
+
 find_package(pybind11 REQUIRED)
 pybind11_add_module(${PROJECT_NAME} gbsbind.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE nlopt)
 
-if ( MSVC )
```

Comment:
This is required to run tests smoothly on windows

Reply:
I do not need this to build with msvc

Reply:
It's true but it's required for running test without installing directly in the conda env during developpement